spec/lib/feature_management_spec.rb

```diff
@@ -181,49 +181,26 @@
     end
   end
 
-  describe '.no_pii_mode?' do
-    let(:proofing_vendor) { :mock }
-    let(:enable_identity_verification) { false }
-
-    before do
-      allow_any_instance_of(Figaro.env).to receive(:profile_proofing_vendor).
-        and_return(proofing_vendor)
-      allow(Figaro.env).to receive(:enable_identity_verification).
-        and_return(enable_identity_verification.to_json)
-    end
-
-    subject(:no_pii_mode?) { FeatureManagement.no_pii_mode? }
-
-    context 'with mock ID-proofing vendors' do
-      let(:proofing_vendor) { :mock }
-
-      context 'with identity verification enabled' do
-        let(:enable_identity_verification) { true }
-
-        it { expect(no_pii_mode?).to eq(true) }
-      end
-
-      context 'with identity verification disabled' do
-        let(:enable_identity_verification) { false }
+  describe '.fake_banner_mode?' do
+    subject(:fake_banner_mode?) { FeatureManagement.fake_banner_mode? }
 
-        it { expect(no_pii_mode?).to eq(false) }
+    context 'fake banner mode in production' do
+      it 'returns true' do
+        allow(Figaro.env).to receive(:domain_name).
+          and_return('secure.login.gov')
+        allow(Rails.env).to receive(:production?).
+          and_return(true)
+        expect(FeatureManagement.fake_banner_mode?).to eq(false)
       end
     end
 
-    context 'with real ID-proofing vendors' do
-      let(:proofing_vendor) { :not_mock }
-
-      context 'with identity verification enabled' do
-        let(:enable_identity_verification) { true }
-
-        it { expect(no_pii_mode?).to eq(false) }
-      end
-
-      context 'with identity verification disabled' do
-        let(:enable_identity_verification) { false }
-
-        it { expect(no_pii_mode?).to eq(false) }
-      end
+    context 'fake Banner mode in test'
+    it 'returns true' do
+      allow(Figaro.env).to receive(:domain_name).
+        and_return('test.login.gov')
+      allow(Rails.env).to receive(:production?).
+        and_return(true)
+      expect(FeatureManagement.fake_banner_mode?).to eq(true)
```

Comment:
Can we also please add a third test for this scenario?

```ruby
context 'when the host is not secure.login.gov and Rails env is not production' do
  it 'does not display the fake banner' do
```

Reply:
DONE